static/js/index.js

```diff
@@ -80,6 +81,18 @@ ep_comments.prototype.init = function(){
 
   utils.getPadInner().find('#innerdocbody').addClass('comments');
 
+  this.api.setHandleCommentCreation(function() {
+    var rep, innerAce;
+    ace.callWithAce(function(ace) {
+      innerAce = ace;
+      rep = innerAce.ace_getRep();
+    },'thisEventWillNotBeCaptured', true);
+
+    var caretLine = rep.selStart[0];
+    selectLine.selectLineIfThereIsNoTextSelected(innerAce, rep, caretLine);
```

Comment:
sugestão: fazer o último argumento da `selectLineIfThereIsNoTextSelected` ser opcional, e usar `rep.selStart[0]` como `caretLine` se ele não for fornecido. Vai deixar o código mais limpo tb